manticore/__main__.py

```diff
@@ -29,6 +29,9 @@ def main() -> None:
     """
     Dispatches execution into one of Manticore's engines: evm or native.
     """
+    # Only print with Manticore's logger
+    logging.getLogger().handlers = []
```

Comment:
Yes it will. I was assuming nobody would ever call this function outside of running manticore from the command line, which means they wouldn't be controlling the logger programmatically anyway.

However, I think a better place for this line would be
```python3
if __name__ == "__main__":
    logging.getLogger().handlers = []
    main()
```
What do you think?

Reply:
That'd be perfect!